src/safeds/ml/classical/classification/_support_vector_machine.py

```diff
@@ -13,14 +13,30 @@
 
 
 class SupportVectorMachine(Classifier):
-    """Support vector machine."""
+    """
+    Support vector machine.
 
-    def __init__(self) -> None:
+    Parameters
+    ----------
+    c: float
+        The strength of regularization
+
+    Raises
+    ------
+    ValueError
+        If the strength of regularization is less than or equal to 0.
```

Comment:
Since `c` is quite different from "strength of regularization" it might be better to write this:

```suggestion
        If `c` is less than or equal to 0.
```